src/pages/GetHelp.vue

```diff
@@ -3,172 +3,213 @@
     <div class="wrapper">
       <div v-if="error" class="error">{{ error }}</div>
       <h1 class="q-px-lg uppercase">{{ $t('getHelp') }}</h1>
+      <q-form @submit="send()">
```

Comment:
I had encountered issues using @submit="send()" on the login form.
Just saying, you shouldn't invoke the send function when you pass it to submit.
But even bette, don't use @submit="send" on the q-form but rather $click.prevent="send" on the send button.

Reply:
Okay, will change that accordingly